Decred Wallet/Features/Security/SecurityViewController.swift

```diff
@@ -48,7 +48,6 @@ class SecurityViewController: SecurityBaseViewController {
     override func viewDidLoad() {
         self.btnPassword.setBackgroundColor(UIColor.appColors.transparentThinGray, for: .highlighted)
         self.btnPin.setBackgroundColor(UIColor.appColors.transparentThinGray, for: .highlighted)
-        
```

Comment:
This space is necessary. Signifies that the code above and code below are not completely related.